_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
+members of the community gathered in Boston, over 650 virtual attendees
+joined us worldwide to learn from our expert speakers as they discussed
+topics such as table formats, enhancements and optimizations, and use
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}
```

Comment:
Ethan gave a look into development on Hudi and Trino's Hudi connector, explaining multi-modal indexing and how it can improve query performance. He also gave an overview of the roadmap and future of the connector.